tests/test_export.py

```diff
@@ -254,57 +247,3 @@ def test_export_with_category_path_failure(self):
                 exporter.export()
 
 
-class SanitizeHtmlTests(unittest.TestCase):
```

Comment:
It's verging on overkill but I like making percentages go up.  Let me look at it this weekend.